app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -39,25 +38,16 @@ class LockboxApplication : Application() {
     override fun onCreate() {
         super.onCreate()
         if (leakCanary()) return
+        if (isUnitTest()) return
```

Comment:
for some reason, in the new Robolectric world this onCreate method gets called every time. I had to include this call to keep the real versions of the .shared Stores from getting allocated and accessing things outside of the scope of the test.